zh_cn.lang

```diff
@@ -0,0 +1,591 @@
+gui.xaero_category_duplicate_confirm=重复以下类别？
+gui.xaero_reverse_entity_radar=倒置雷达渲染顺序
+gui.xaero_adjust_height_for_carpetlike_blocks=调整矮方块的Y
+gui.xaero_box_adjust_height_for_carpetlike_blocks=对于地毯型方块（例如地毯、单层雪、睡莲）的高度值减1。这可以避免这些方块像完整方块那样在地图上显示明显的阴影。
+gui.xaero_box_category=查看/编辑 §e%1$s
+gui.xaero_box_category_settings=§e%1$s§r 的设置
+gui.xaero_auto_convert_wp_distance_km=将距离转化为KM
+gui.xaero_auto_convert_wp_distance_km_never=从不
+gui.xaero_waypoint_distance_precision=路径点距离精度
+gui.xaero_sort_angle=角度距离
+gui.xaero_toggle_enabled=启用
+gui.xaero_toggle_disabled=禁用
+gui.xaero_main_entity_dot_size=主要实体点大小
+gui.xaero_category_settings_save_confirm=退出前保存更改？
+gui.xaero_category_settings_save_confirm_warning=警告：此配置屏幕近期被替换！
+gui.xaero_start_fading_at=深度衰减的最小Y距离
+gui.xaero_box_start_fading_at=“显示实体深度”设置启用时，实体开始在雷达上变暗的最小Y轴坐标差。
+gui.xaero_start_fading_at_auto=自动
+gui.xaero_legacy_color=旧版
+gui.xaero_partial_y_teleportation=tp时，使用y的相对距离
+gui.xaero_box_partial_y_teleportation=传送至一个路径点或者地图位置时，将Y坐标向上偏移0.5，以避免从地毯类方块跌落至虚空。否则，直接使用整数的Y坐标。
+gui.xaero_delete_reached_deathpoints=删除已到达的死亡点
+gui.xaero_box_delete_reached_deathpoints=距离死亡点4个方块以内时自动删除该死亡点。这需要死亡点在游戏内或小地图上可见。死亡点在创建后的5秒内会被保护以防删除。
+gui.xaero_hide_minimap_under_screen=在屏幕下隐藏
+gui.xaero_box_hide_minimap_under_screen=启用了GUI屏幕（例如物品栏）时隐藏小地图，除了与小地图有关的屏幕以及聊天屏幕。
+gui.xaero_north_compass_color=北罗盘颜色
+gui.xaero_north_compass_color_default=默认
+gui.xaero_category_add_to_list_custom=确认自定义输入：%1$s
+gui.xaero_box_category_list_add=支持带有特殊字符 * ) ( 和 | 的实体标识符或匹配模式。\n \n * - 匹配所有。 \n )( - 括号，主要用于 |. \n | - “或”运算符，以匹配相同位置的不同序列。 \n \n 例如，\n minecraft:creeper \n minecraft:zombie(_villager|_horse) - 僵尸村民和僵尸马。 \n minecraft:a*l - 任何以'a'开头、'l'结尾的原版实体。
+gui.xaero_using_custom_subworld=路径点：%1$s
+gui.xaero_hide_minimap_under_f3=F3下隐藏
+gui.xaero_box_hide_minimap_under_f3=启用了F3调试屏幕时，隐藏小地图。
+gui.xaero_entity_display_y=显示 Y 坐标
+gui.xaero_entity_display_y_full=显示实体的 Y 坐标
+gui.xaero_block_light_value=方块光：%d
+gui.xaero_sky_light_value=天空光：%d
+gui.xaero_all_light_value=光：%d
+gui.xaero_both_light_value=方块光：%d 天空光：%d
+gui.xaero_toggle_manual_cave_mode=启用手动洞穴模式
+gui.xaero_manual_cave_mode=手动洞穴模式
+gui.xaero_manual_cave_mode_start=手动洞穴模式起始点Y值
+gui.xaero_box_manual_cave_mode_start=启用了手动洞穴模式时，从这个Y值开始映射方块。手动洞穴模式可以用按键绑定启用。对于“自动”选项，一直走左边。
+gui.xaero_manual_cave_mode_start_auto=自动
+gui.xaero_chunk_grid_line_width=区块网线条宽度
+gui.xaero_radar_render_over_minimap=在小地图框架之上渲染
+gui.xaero_box_radar_render_over_minimap=在小地图框架的上方渲染实体，而不是渲染在框架内。这样渲染的实体将会像路径点那样，超出范围时贴在边框处。可见的实体依然仅限于仅在游戏客户端中加载了的。\n \n 选项“玩家列表” — 仅在按下了“列举玩家”按键（默认为Tab）时渲染小地图。
+gui.xaero_radar_render_radar_over_frame=在框架之上渲染雷达
+gui.xaero_radar_over_map_never=从不
+gui.xaero_radar_over_map_list=玩家列表
+gui.xaero_radar_over_map_always=总是
+gui.xaero_compass_over_everything=指南针置于顶层
+gui.xaero_box_compass_over_everything=将指南针字母（N、E、S、W）渲染在所有地图元素上，而不是渲染在路径点等元素下方。
+gui.xaero_waypoint_server_shared=服务器
+gui.xaero_pac_players=显示派对/盟友玩家
+gui.xaero_box_pac_players=当有与你分享了坐标（由Open Parties and Claims模组）的玩家超出小地图框架边界或因其他原因不可见时，显示这些玩家。
```

Comment:
Should be "who are sharing their coordinates with you". Translates back weirdly.